contrib/create-release-candidate.sh

```diff
@@ -0,0 +1,287 @@
+#! /usr/bin/env bash
```

Comment:
Are there any significant diffs with this script between what Fluo and Accumulo use?

Reply:
No. I copied from fluo-recipes and edited as needed.